rocAL_pybind/amd/rocal/types.py

```diff
@@ -103,6 +103,11 @@
 from rocal_pybind.types import FT
 from rocal_pybind.types import TF
 
+#     RocalOutOfBoundsPolicy
+from rocal_pybind.types import PAD
+from rocal_pybind.types import TRIMTOSHAPE
+from rocal_pybind.types import ERROR
```

Comment:
In line 36, the int32 and uint32 types are not there is NSR working?

Reply:
since those are used in tensor.cpp